datastore/datastore_test.go

```diff
@@ -1413,6 +1423,22 @@ var testCases = []testCase{
 		"",
 		"",
 	},
+	{
+		"embedded time field",
+		&SpecialTime{MyTime: EmbeddedTime{now}},
+		&SpecialTime{MyTime: EmbeddedTime{now}},
+		"",
+		"",
+	},
+	{
+		"embedded time load",
+		&PropertyList{
+			Property{Name: "MyTime.", Value: now, NoIndex: false, Multiple: false},
```

Comment:
Should it maybe be encoded as "MyTime.Time"? Or is it just not possible to change the encoder now?